### DISCUSSION TITLE
New user here, how can I change the navbar on every site (home, doc and blog)?

### DISCUSSION THREAD

**Discussion by itsme-microcip** [thuliteio/doks#1312] [Q&A] [answered]
I want to change the link associated with the GitHub icon in my project. However, whenever I update the link and then restart the development environment using npm run dev, the link reverts to its original value. How can I make the change permanent? Which file I have to edit to for example add new things on the navbar

Thanks in advance!

**Comment by h-enk** [comment, marked as answer]
Hello @FedeP14 Put your social links in the `[[social]]` section of `config/_default/menus/menus.en.toml`

To clear Hugo's server cache, run for example:

```bash
npx shx rm -rf public resources hugo_stats.json .hugo_build.lock
```
